### PR TITLE
New syntax [Inductive Acc A R | x : Prop := ...]

### PR DESCRIPTION
**Pull request by SkySkimmer**
to control uniform parameters.

This replaces the attribute.

### CODE REVIEW

File: doc/sphinx/language/gallina-specification-language.rst
Comment (Zimmi48) [inline doc/sphinx/language/gallina-specification-language.rst]:
```suggestion
     beginning (flag is unset) or at the end of the parameters (flag is set) when not explicitly given.
```

**Review by Zimmi48 — APPROVED** [review]
Doc looks good, just a small clarification suggestion.

**Comment by Zimmi48** [comment]
@coq/vernac-maintainers It would be great not to delay this PR too long since it reverts a recently introduced syntax.

**Comment by ejgallego** [comment]
I can merge this, and the PR looks OK but I am not really specialist on inductives subtleties, maybe @mattam82 @ppedrot @herbelin should approve this?

**Review by maximedenes — APPROVED** [review]
This is good, and indeed makes more sense than a binary attribute.

**Comment by ejgallego** [comment]
I'd say let's merge before the end of the week then if no further comments.

**Review by ppedrot — APPROVED** [review]

**Comment by herbelin** [comment]
OK to me.

**Pull request merged**
